src/Game/AI/AI/aiActorWaterDepthSelect.cpp

```diff
@@ -24,4 +64,13 @@ void ActorWaterDepthSelect::loadParams_() {
     getStaticParam(&mForceDeepChange_s, "ForceDeepChange");
 }
 
+inline bool ActorWaterDepthSelect::isDeep() const {
```

Comment:
```suggestion
bool ActorWaterDepthSelect::isDeep() const {
```
This doesn't need to be marked `inline`, and marking a function `inline` does not necessarily mean the compiler will inline it. Conversely, the compiler can still choose to inline a function that has *not* been marked inline - and I'd expect it to do so here.